src/app/work-item/work-item.service.ts

```diff
@@ -57,6 +57,24 @@ export class WorkItemService {
       .catch(this.handleError);
   }
 
+ getWorkItemTypes(): Promise<any[]> {
+    this.logger.log('WorkItemService using url ' + this.workItemTypeUrl);
```

Comment:
Sudipta requested an additional log message when a URL other than that used for the work items was accessed.